yarn.lock

```diff
@@ -2889,7 +2889,7 @@ __metadata:
     debug: "npm:^4.3.1"
     find-up: "npm:~5.0.0"
     getenv: "npm:^1.0.0"
-    glob: "npm:7.1.6"
+    glob: "npm:^9.0.0"
```

Comment:
```suggestion
    glob: "npm:7.1.6"
```

Reply:
For some reason github won't let me commit this here, saying "the diff has recently been update". I will push a commit that makes these changes.

Reply:
012944bf4e